Makefile

```diff
@@ -79,6 +84,13 @@ else
 	cd viam-cartographer && ./scripts/build_cartographer.sh && ./scripts/build_viam_cartographer.sh
 endif
 
+build-debug: build-module
+ifneq ($(wildcard viam-cartographer/cartographer/build/.),)
+	cd viam-cartographer && ./scripts/build_viam_cartographer_debug.sh 
```

Comment:
Added so that we can get debug symbols so we can run our unit tests with GDB if we get segfaults, etc in development.